frontend/src/metabase/sharing/components/PulsesListSidebar.jsx

```diff
@@ -79,54 +91,103 @@ function PulsesListSidebar({
-  const [first, ...rest] = recipients;
+function buildRecipientText(pulse) {
+  const {
+    channels: [firstChannel],
+  } = pulse;
+  if (firstChannel.channel_type !== "email") {
+    return "";
+  }
+
+  const [firstRecipient, ...otherRecipients] = firstChannel.recipients;
+  const firstRecipientText = firstRecipient.common_name || firstRecipient.email;
+  return _.isEmpty(otherRecipients)
+    ? firstRecipientText
+    : `${firstRecipientText} ${ngettext(
+        msgid`and ${otherRecipients.length} other`,
+        `and ${otherRecipients.length} others`,
+        otherRecipients.length,
+      )}`;
+}
 
-  let text = "";
+function buildFilterText(pulse, parameters) {
+  const activeParameters = getActivePulseParameters(pulse, parameters);
 
-  if (rest != null && rest.length > 0) {
-    text += ngettext(
-      msgid` and ${rest.length} other`,
-      ` and ${rest.length} others`,
-      rest.length,
-    );
+  if (_.isEmpty(activeParameters)) {
+    return "";
   }
 
+  const [firstParameter, ...otherParameters] = activeParameters;
+  const numValues = [].concat(firstParameter.value).length;
+  const firstFilterText = `${firstParameter.name} is ${
+    numValues > 1 ? t`${numValues} selections` : firstParameter.value
+  }`;
+
+  return _.isEmpty(otherParameters)
+    ? firstFilterText
+    : `${firstFilterText} ${ngettext(
+        msgid`and ${otherParameters.length} more filter`,
+        `and ${otherParameters.length} more filters`,
+        otherParameters.length,
+      )}`;
+}
+
+function PulseDetails({ pulse, parameters }) {
+  const recipientText = buildRecipientText(pulse);
+  const filterText = buildFilterText(pulse, parameters);
+
   return (
     <div className="text-medium hover-child">
       <ul
-        className="flex flex-wrap scroll-x scroll-y"
+        className="flex flex-column scroll-x scroll-y text-unspaced"
         style={{ maxHeight: 130 }}
       >
-        <li className="flex align-center mr1 text-bold text-medium hover-child hover--inherit">
-          <Icon
-            name="group"
-            className="text-medium hover-child hover--inherit"
-            size={12}
-          />
-          <span
-            className="ml1 text-medium hover-child hover--inherit"
-            style={{ fontSize: "12px" }}
-          >
-            {first.common_name || first.email}
-            {text !== "" && text}
-          </span>
-        </li>
+        {recipientText && (
+          <li className="flex align-center mr1 text-bold text-medium hover-child hover--inherit">
+            <Icon
+              name="group"
+              className="text-medium hover-child hover--inherit"
+              size={12}
+            />
+            <span
+              className="ml1 text-medium hover-child hover--inherit"
+              style={{ fontSize: "12px" }}
+            >
+              {recipientText}
+            </span>
+          </li>
+        )}
+        {filterText && (
```

Comment:
Maybe this could be replaced with `{...PLUGIN_PULSE_DETAILS.map(f => f(props))}` or something.